<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>519</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>669</width>
    <height>519</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>669</width>
    <height>519</height>
   </size>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>651</width>
     <height>451</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="general">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_General">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>631</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <widget class="QCheckBox" name="checkBox_option1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Open URL in Web Browser after playlist was generated</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_option2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy URL to clipboard after playlist was generated</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_language">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>631</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <widget class="QComboBox" name="comboBox_language">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>English</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Deutsch</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_theme">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>631</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Theme</string>
     </property>
     <widget class="QRadioButton" name="radioButton_OS">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>OS Standard</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_white">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>White</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_dark">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dark</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="shortcuts">
    <attribute name="title">
     <string>Keyboard Shortcuts</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_keyboard_shortcuts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>631</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string>Keyboard Shortcuts</string>
     </property>
     <widget class="QKeySequenceEdit" name="keySequenceEdit_option1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option1_text">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>42</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Import new playlist</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option2_text">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>82</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Export playlist</string>
      </property>
     </widget>
     <widget class="QKeySequenceEdit" name="keySequenceEdit_option2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option3_text">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>122</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QKeySequenceEdit" name="keySequenceEdit_option3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option4_text">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>162</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QKeySequenceEdit" name="keySequenceEdit_option4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>42</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Cmd+I</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>82</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Cmd+E</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>122</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Alt+R</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>162</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Cmd+G</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_change_option1">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_change_option2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_change_option3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_change_option4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>160</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>202</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Alt+S</string>
      </property>
     </widget>
     <widget class="QKeySequenceEdit" name="keySequenceEdit_option5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_keyboard_shortcuts_option5_text">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>202</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Shuffle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_change_option5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>200</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
